README.md

```diff
@@ -1,79 +1,60 @@
 # angular-cli-esri-map
 
-This tutorial shows you how to integrate the [ArcGIS API for JavaScript](https://developers.arcgis.com/javascript), which is an enterprise geospatial API, using [Angular CLI](https://github.com/angular/angular-cli).
+This branch contains a simple but complete application that uses the ArcGIS API for JavaScript, an enterprise geospatial API, and Angular CLI. It uses esri-loader, a small library that helps load ArcGIS JavaScript API modules (v3.x or v4.x) in non-Dojo applications.
 
-This repo demonstrates implementing a simple, extensible mapping component. Use this repo as a building block for adding in more advanced capabilities.
+To use arcgis-webpack-plugin instead of esri-loader to load the ArcGIS API for JavaScript modules, check out the `@TODO ADD BRANCH NAME` branch of this repo.
```

Comment:
Just a reminder to myself that we'll need to up the `branch` name.

Reply:
Let me know if you'd like me to change this to an actual branch name before this gets merged in 😎